### PR TITLE
[ui] Add filter selector to the search box

### PR DESCRIPTION
**Pull request by evamillan**
Adds an optional selector that displays the list of valid filters next to the search input. When a filter is selected from the list, it is added to the input value.

### CODE REVIEW

File: ui/src/components/Search.vue [hunk 1]
```diff
@@ -175,23 +209,47 @@ export default {
         vfilter => vfilter.filter === filter
       );
       return validFilter.type === "boolean";
+    },
+    setFilter(item) {
```
Comment (sduenas) [inline ui/src/components/Search.vue:213]: We need some unit tests to check that when any the possible filters are clicked, the input is correctly set.

File: ui/src/components/Search.vue [hunk 2]
```diff
@@ -9,20 +9,49 @@
...
   >
-    <template v-slot:message="{ key, message }">
-      <span>{{ message }}</span>
-      <v-btn href="/search-help" target="_blank" color="primary" text x-small>
-        View search syntax
-        <v-icon x-small right>
-          mdi-help-circle-outline
-        </v-icon>
-      </v-btn>
+    <template v-slot:prepend v-if="filterSelector">
```
Comment (sduenas) [inline ui/src/components/Search.vue:21]: Can we improve the style of the search box?. In my opinion, it's very big if you compare it with other components of the page. In fact its size is the same or similar of the `Indiviuals` and `Organizations` table. Maybe reducing the size and the font might work.

**Review by sduenas — CHANGES_REQUESTED** [review]
In overall works great but I think we need to improve some parts. What do you think?

File: ui/src/components/Search.vue (hunk 1 above)
Comment (evamillan) [reply ui/src/components/Search.vue:214]: I added the unit tests on `search.spec.js`.

File: ui/src/components/Search.vue (hunk 2 above)
Comment (evamillan) [reply ui/src/components/Search.vue:22]: I reduced the font and the overall size of the box a bit, let me know if I can do anything else to improve it further.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**